### PR TITLE
Add `active_child_template` to universal media player

### PR DESCRIPTION
**Pull request by Drafteed**
## Proposed change

Docs related to core PR: https://github.com/home-assistant/core/pull/88816
I also slightly formatted the example sections.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/88816
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Review by emontnemery — CHANGES_REQUESTED** [review]
The name of the new parameter, `active_children_template` suggests plural, but the documentation suggests singular, i.e. a single active child:
> Template must return the `entity_id` of the child that will be selected as active

Which one is it?

File: source/_integrations/universal.markdown [hunk 1]
```diff
@@ -99,6 +103,8 @@ unique_id:
 
 The Universal Media Player will primarily imitate one of its `children`. The Universal Media Player will control the first child on the list that is active (not idle/off). The Universal Media Player will also inherit its state from the first active child if a `state_template` is not provided. Entities in the `children:` list must be media players, but the state template can contain any entity.
 
+Using `active_child_template` will allow you to specify an active entity if the default behavior is not suitable for your task. Template must return the `entity_id` of the child that will be selected as active, or `None` to return the default behavior.
```
Comment (frenck) [inline source/_integrations/universal.markdown]:
```suggestion
Using `active_child_template` will allow you to specify an active entity if the default behavior is unsuitable for your task. The template must return the `entity_id` of the child that will be selected as active or `None` to return the default behavior.
```

File: source/_integrations/universal.markdown [hunk 2]
```diff
@@ -71,6 +71,10 @@ children:
   description: Ordered list of child media players that this entity will control.
   required: false
   type: list
+active_child_template:
+  description: "A [template](/docs/configuration/templating/) that will allow to select (override) active child. Must return the `entity_id` of the child that will be selected as active, or `None` to use the default behavior."
```
Comment (frenck) [inline source/_integrations/universal.markdown]:
```suggestion
  description: "A [template](/docs/configuration/templating/) that will allow to select (override) active child. Must return the `entity_id` of the child selected as active, or `None` to use the default behavior."
```

**Review by frenck — APPROVED** [review]
Thanks, @Drafteed 👍 

../Frenck

**Pull request merged**